packages/core/core-flows/src/product/workflows/update-products.ts

```diff
@@ -166,14 +167,15 @@ export const updateProductsWorkflow = createWorkflow(
 
     const toDeleteLinks = transform({ currentLinks }, prepareToDeleteLinks)
 
-    dismissRemoteLinkStep(toDeleteLinks)
-
     const salesChannelLinks = transform(
       { input, updatedProducts },
       prepareSalesChannelLinks
     )
 
-    createRemoteLinkStep(salesChannelLinks)
+    parallelize(
```

Comment:
suggestion: Can you check that this is being executed, I believe I had the case that if you dont store the result in a variable used elsewhere this does not get executed :D I d like to get your feedback

Reply:
just reverted it